core/quality/morqa_transport.go

```diff
@@ -102,6 +105,27 @@ func sessionDataToMetricsEvent(context sessionDataContext) *metrics.Event {
 					ServiceType:    context.ServiceType,
 					ProviderContry: context.ProviderCountry,
 					ConsumerContry: context.ConsumerCountry,
+					AccountantId:   context.AccountantID,
+				},
+			},
+		},
+	}
+}
+
+func sessionTokensToMetricsEvent(context sessionTokensContext) *metrics.Event {
```

Comment:
It can be  shorter `ctx sessionTokensContext` and will not collide with context package.

Reply:
Trouble of future, not colliding. Renamed anyways.